recipes/tachyon/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.99b6" %}
+{% set name = "tachyon" %}
+{% set md5 = "9e5ed4045c472e01e57480d7c7eee849" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://pkgs.fedoraproject.org/repo/pkgs/{{ name }}/{{ name }}-{{ version }}.tar.gz/{{ md5 }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
+  patches:
+    # Patches taken from https://github.com/sagemath/sage/tree/develop/build/pkgs/tachyon/patches
+    # Use cc from environment
+    - patches/Make-arch.patch
+    # Use libpng
+    - patches/Make-config.patch
+    # Use LDFLAGS from environment
+    - patches/Makefile.patch
```

Comment:
Did Sage try submitting these upstream? Is there an upstream still to submit them too?